Gemfile

```diff
@@ -14,7 +14,7 @@ gem 'active_job_status', '~> 1.2.1'
 gem 'bagit'
 gem 'bcrypt_pbkdf', '~> 1.1' # Needed to support more secure ssh keys
 gem 'bixby', '~> 3.0'
-gem "blacklight_range_limit", '~>6'
+gem "blacklight_range_limit", '~>7.0.0'
```

Comment:
I don't think blacklight_range_limit 7.x is compatible with blacklight 6.x 

Cypripedium has to run blacklight 6 (because of Hyrax) 
https://github.com/curationexperts/cypripedium/blob/v5.1.1/Gemfile.lock#L118

See note about syncrhonizing versions here https://github.com/projectblacklight/blacklight_range_limit#installation

Can your remove this or confirm that you have tested against production-like date data?